commit ad6965288ceaa6cb1f101264219503ca93010e53
Author: Nick Danyluk <ndanyluk7@gmail.com>
Date:   Wed Oct 7 19:36:02 2020 -0400

    Bump META Linux-Maximum to kernel 5.8 (#11019)
    
    Bump max kernel version to 5.8 to match the supported releases
    indicated in the release notes of 0.8.5
    
    Signed-off-by: Nick Danyluk <ndanyluk7@gmail.com>
    Reviewed-by: George Melikov <mail@gmelikov.ru>
    Reviewed-by: Tony Hutter <hutter2@llnl.gov>

diff --git a/META b/META
index fa106fe1b..ff0443278 100644
--- a/META
+++ b/META
@@ -6,5 +6,5 @@ Release:       1
 Release-Tags:  relext
 License:       CDDL
 Author:        OpenZFS on Linux
-Linux-Maximum: 5.6
+Linux-Maximum: 5.8
 Linux-Minimum: 2.6.32
